covid19_sfbayarea/data/san_francisco.py

```diff
@@ -154,11 +153,11 @@ def get_age_table(session : SocrataApi, resource_ids: Dict[str, str]) -> List[Di
                 "51_to_60": "51-60", "61_to_70": "61-70", "71_to_80": "71-80", "81_and_older": "81+",}
 
 
-    params = {'$select': 'age_group, sum(confirmed_cases)', '$order': 'age_group', '$group': 'age_group'}
+    params = {'$select': 'age_group, MAX(cumulative_confirmed_cases) as cumul_cases', '$group': 'age_group'}
```

Comment:
If the source provides a cumulative number, I'd rather use it directly than calculating the sum over the timeseries. I agree this is a bit weird, and over in the new `get_gender_table` I use a technique that makes more sense of grabbing the most recent entries. 

I guess I'm wary that the age groups might be more likely to change, and we've been discussing doing away with age group rekeying altogether. I think this this technique of grabbing the max cumulative totals by age group offers the most flexibility, and grabs the totals as directly reported by the county.

Reply:
Makes sense! Thinking about this point especially:

> I'm wary that the age groups might be more likely to change, and we've been discussing doing away with age group rekeying altogether. I think this this technique of grabbing the max cumulative totals by age group offers the most flexibility

Makes me feel like this approach is better than the one you’re using in `get_gender_table`, on the chance that a fourth (or more) gender categorizations could be added. (I don’t expect that’s likely, though, and this is NOT a statement that you should change that code before merging — unless you agree and want to. It’s in now way a show-stopper. 😄)

Reply:
I changed it to parallel the `get_age_table` method. Since we want to be stricter about gender groups in case we need to re-aggregate genders called out as trans, I did a strict check that the genders received in the data are the same as what we expect in `GENDER_KEYS`. And, I learned that `dict.keys()` returns a set-like collection!